wp-content/themes/inf_theme/admin/class-admin.php

```diff
@@ -71,7 +49,7 @@ public function enqueue_scripts() {
    * @param string $color_scheme Color scheme string.
    * @return string              Modified color scheme.
    *
-   * @since 2.1.0
+   * @since 1.0.0
    */
   public function set_admin_color_based_on_env( $color_scheme ) {
```

Comment:
Not sure we have this in the wiki, but it should be added, explaining what it does and why.

Reply:
We don't have a lot of stuff in wiki, I will update this at some point

Reply:
Or maybe @IvanGrginovInf can, since he has the gift of really writing things out in detail 😄😛 

seeWhatIDidThere xD

Reply:
It's never too late to contribute :D